From 1de7f30668339f2bf3d74f1f1d19dd7714f8ff4d Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Mon, 22 Aug 2016 17:47:05 +0200
Subject: [PATCH] openssh: skip privilege separation tests

In MSYS2's context, it does not make sense, as we do not have a
special-purpose 'sshd' user account and also no easy way to make one.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 regress/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/regress/Makefile b/regress/Makefile
index 08fd82d..00cff71 100644
--- a/regress/Makefile
+++ b/regress/Makefile
@@ -1,6 +1,6 @@
 #	$OpenBSD: Makefile,v 1.88 2016/06/03 04:10:41 dtucker Exp $
 
-REGRESS_TARGETS=	unit t1 t2 t3 t4 t5 t6 t7 t8 t9 t10 t11 t12 t-exec
+REGRESS_TARGETS=	unit t1 t2 t3 t4 t5 t6 t7 t8 t9 t10 t11 t12
 tests:		prep $(REGRESS_TARGETS)
 
 # Interop tests are not run by default
-- 
2.9.1

